numpy/distutils/fcompiler/gnu.py

```diff
@@ -177,10 +191,12 @@ def get_libgfortran_dir(self):
         libgfortran_dir = None
         if libgfortran_name:
             find_lib_arg = ['-print-file-name={0}'.format(libgfortran_name)]
-            status, output = exec_command(
-                self.compiler_f77 + find_lib_arg, use_tee=0)
-            if not status:
-                libgfortran_dir = os.path.dirname(output)
+            try:
+                output = subprocess.check_output(
+                                       self.compiler_f77 + find_lib_arg)
+                libgfortran_dir = os.path.dirname(str(output))
```

Comment:
Isn't the same historical processing used here? Perhaps you should add a `filepath_from_subprocess_output` function that you use in all these places, and you can stick the comments in. Something like
```
"""
Convert `bytes` in the encoding used by a subprocess into a filesystem-appropriate `str`.

Inherited from `exec_command`, and possibly incorrect.
"""
```

Reply:
I don't see any reason why these two cases would be different - both are asking the same executable for a filename